.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Make sure there is a new line <kbd>enter</kbd> at the end of (almost) every soure file.

Check https://github.com/Gerhut/koa-http-auth/blob/master/.editorconfig maybe you need a editorconfig plugin in your editor